packages/atomic/src/components/search/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -133,6 +134,10 @@ export class AtomicCategoryFacet
    * Also determines the number of additional values to request each time more values are shown.
    */
   @Prop({reflect: true}) public numberOfValues = 8;
+  /**
+   * The tabs on which to display the facet.
+   */
+  @Prop({reflect: true}) public tabs: string = '';
```

Comment:
This should be an `@ArrayProp`, like we do for `atomic-search-interface` -> `fieldsToInclude` for example.

We should not rely on character delimiter for such type of properties, even though it might seem more elegant: They tend to cause other type of issues. 

It's uglier, but "safer" to instead expect implementer to pass in a JSON array... And also, more uniform with the rest.